Assets/VRTK/Source/Scripts/Utilities/SDK/VRTK_SDKSetupSwitcher.cs

```diff
@@ -79,6 +82,14 @@ protected virtual void OnDisable()
         protected virtual void OnLoadedSetupChanged(VRTK_SDKManager sender, VRTK_SDKManager.LoadedSetupChangeEventArgs e)
         {
             Show(ViewingState.Status);
+            if (playareaSync && currentPlayarea != null)
+            {
+                Transform newPlayarea = VRTK_DeviceFinder.PlayAreaTransform();
+                newPlayarea.transform.position = currentPlayarea.transform.position;
+                newPlayarea.transform.rotation = currentPlayarea.transform.rotation;
+                newPlayarea.transform.localScale = currentPlayarea.transform.localScale;
```

Comment:
Maybe this should use our helper method to set the global scale instead? Depending on the exact hierarchy the play area game objects are in `localScale` would probably fail to work in some cases.